src/radical/pilot/configs/resource_anl.json

```diff
@@ -79,14 +81,20 @@
         "agent_config"                : "default",
         "agent_scheduler"             : "CONTINUOUS",
         "agent_spawner"               : "POPEN",
-        "agent_launch_method"         : "SSH",
-        "task_launch_method"          : "MPIRUN",
-        "mpi_launch_method"           : "MPIRUN",
+        "launch_methods"              : {
+                                         "order" : ["MPIRUN", "SSH"],
+                                         "MPIRUN": {
+                                             "pre_exec_cached": [
+                                                 "module use  /soft/modulefiles",
+                                                 "module load openmpi/4.1.1-gcc"
```

Comment:
Just for future reference, MPICH is expected to be the primary MPI used on Aurora.